presto-server-rpm/src/test/java/io/prestosql/server/rpm/ServerIT.java

```diff
@@ -63,6 +63,8 @@ private static void testServer(String baseImage, String rpmHostPath, String expe
         String command = "" +
                 // install RPM
                 "yum localinstall -q -y " + rpm + "\n" +
+                // temporary hack to allow Java 8
+                "echo '-Dpresto-temporarily-allow-java8=true' >> /etc/jvm.config\n" +
```

Comment:
This should be applied only when running on `centos7-oj8` image

Reply:
There's no harm in having it for both -- it's ignored on Java 11. Do you have a specific concern with having it for both?

Reply:
I thought that we could verify this property is _not_ needed when running on 11.

Reply:
Make sense. I'll update it.